### ISSUE TITLE
Icons on bottom tab nav should all be the same height/proportion

### ISSUE THREAD

**Issue by bsteinbeigle** [Methodician/guytime#88]

**Comment by bsteinbeigle** [comment]
Yes please! the "browse" profile icon is too large. I'll take any suggestions on a different icon as well. I think the dude with a sideways hat is fairly prescriptive.

**Comment by kaxline** [comment]
@bsteinbeigle Since we're using the Material UI framework, we should use their icon set. Here's a query for 'user' icons:

https://fonts.google.com/icons?selected=Material+Icons&icon.query=user

**Comment by bsteinbeigle** [comment]
@kaxline Thanks for sending! makes sense on sticking with Material UI with exception to the 'man shake' icon. I'll give these a look and let you know what I'm thinking. I don't think we need to spend a whole lot of time on icons at this stage unless they are confusing as you outlined for the 'next profile' icon.
